bycycle/burst/utils.py

```diff
@@ -20,6 +20,13 @@ def check_min_burst_cycles(is_burst, min_n_cycles=3):
     -------
     is_burst : 1d array
         Updated burst array.
+
+    Examples
+    --------
+    Removed bursts with less than 3 consectutive cycles.
+
+    >>> is_burst = np.random.choice([True, False], size=10)
```

Comment:
I think the random array is potentially a little confusing here. 
We should be able to use an exact array, and then show the answer, right?

Like:

```
>>> is_burst = np.array([False, True, True, False, True, True, True, True, False, False ])
>>> is_burst = check_min_burst_cycles(is_burst)
np.array([False, False False, False, True, True, True, True, False, False ])